action.yml

```diff
@@ -42,8 +42,9 @@ runs:
   - name: Test source distribution
     shell: bash
     run: |
+      set -e
```

Comment:
I think this is already the default: https://github.com/OpenAstronomy/build-python-dist/runs/5876701874?check_suite_focus=true#step:3:351

Reply:
Sounds good, I'll remove